client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -398,7 +396,7 @@ function mapStateToProps( state ) {
 
 	const siteId = getSelectedSiteId( state );
 	const productInstallStatus = getJetpackProductInstallStatus( state, siteId );
-	const rewindState = get( getRewindState( state, siteId ), 'state', 'uninitialized' );
+	const rewindState = getRewindState( state, siteId )?.state ?? 'uninitialized';
```

Comment:
Looking at `getRewindState()` it seems like it will always return an object, no? So the first optional chaining seems to be unnecessary IMHO.